client/scripts/com/monsters/maproom_manager/MapRoomManager.as

```diff
@@ -325,7 +325,7 @@ package com.monsters.maproom_manager
          }
          if(MapRoomManager.instance.isInMapRoom2 == false)
          {
-            MapRoom3ConfirmMigrationPopup.instance.Show(true);
+            // MapRoom3ConfirmMigrationPopup.instance.Show(true);
```

Comment:
This seems fine, if we're going to modify the client in any way, this would be an acceptable change.